### PR TITLE
main.cc: disable stall detector for debug ARM builds

### LINKED ISSUE #15090: [aarch64] AddressSanitizer: SEGV on unknown address 0xfffffffffffffff8 in tests on aarch64 debug mode
Seen in https://jenkins.scylladb.com/view/master/job/scylla-master/job/build/1578/artifact/testlog/aarch64/debug/alternator.run.1.log
With scylla version 7275b8967cf5
```
! _pytest.outcomes.Exit: Scylla appears to have crashed in test test_streams.py::test_list_streams_paged_with_new_table !
Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/urllib3/util/connection.py", line 95, in create_connection
    raise err
  File "/usr/lib/python3.11/site-packages/urllib3/util/connection.py", line 85, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/botocore/httpsession.py", line 455, in send
    urllib_response = conn.urlopen(
                      ^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/urllib3/connectionpool.py", line 798, in urlopen
    retries = retries.increment(
              ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/urllib3/util/retry.py", line 525, in increment
    raise six.reraise(type(error), error, _stacktrace)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/urllib3/packages/six.py", line 719, in reraise
    raise value
  File "/usr/lib/python3.11/site-packages/urllib3/connectionpool.py", line 714, in urlopen
    httplib_response = self._make_request(
                       ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/urllib3/connectionpool.py", line 415, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/usr/lib/python3.11/site-packages/urllib3/connection.py", line 244, in request
    super(HTTPConnection, self).request(method, url, body=body, headers=headers)
  File "/usr/lib64/python3.11/http/client.py", line 1286, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/lib/python3.11/site-packages/botocore/awsrequest.py", line 94, in _send_request
    rval = super()._send_request(
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/http/client.py", line 1332, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/lib64/python3.11/http/client.py", line 1281, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.11/site-packages/botocore/awsrequest.py", line 123, in _send_output
    self.send(msg)
  File "/usr/lib/python3.11/site-packages/botocore/awsrequest.py", line 218, in send
    return super().send(str)
           ^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/http/client.py", line 979, in send
    self.connect()
  File "/usr/lib/python3.11/site-packages/urllib3/connection.py", line 205, in connect
    conn = self._new_conn()
           ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <botocore.awsrequest.AWSHTTPConnection object at 0xffff9e5050d0>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/pytest", line 8, in <module>
    sys.exit(console_main())
             ^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/_pytest/config/__init__.py", line 190, in console_main
    code = main()
           ^^^^^^
  File "/usr/lib/python3.11/site-packages/_pytest/config/__init__.py", line 167, in main
    ret: Union[ExitCode, int] = config.hook.pytest_cmdline_main(
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/pluggy/_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/pluggy/_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/pluggy/_callers.py", line 60, in _multicall
    return outcome.get_result()
           ^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/pluggy/_result.py", line 60, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/usr/lib/python3.11/site-packages/pluggy/_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
          ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/_pytest/main.py", line 317, in pytest_cmdline_main
    return wrap_session(config, _main)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/_pytest/main.py", line 305, in wrap_session
    config.hook.pytest_sessionfinish(
  File "/usr/lib/python3.11/site-packages/pluggy/_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/pluggy/_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/pluggy/_callers.py", line 55, in _multicall
    gen.send(outcome)
  File "/usr/lib/python3.11/site-packages/_pytest/terminal.py", line 808, in pytest_sessionfinish
    outcome.get_result()
  File "/usr/lib/python3.11/site-packages/pluggy/_result.py", line 60, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/usr/lib/python3.11/site-packages/pluggy/_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
          ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/_pytest/runner.py", line 106, in pytest_sessionfinish
    session._setupstate.teardown_exact(None)
  File "/usr/lib/python3.11/site-packages/_pytest/runner.py", line 530, in teardown_exact
    raise exc
  File "/usr/lib/python3.11/site-packages/_pytest/runner.py", line 523, in teardown_exact
    fin()
  File "/usr/lib/python3.11/site-packages/_pytest/fixtures.py", line 685, in <lambda>
    subrequest.node.addfinalizer(lambda: fixturedef.finish(request=subrequest))
                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/_pytest/fixtures.py", line 1037, in finish
    raise exc
  File "/usr/lib/python3.11/site-packages/_pytest/fixtures.py", line 1030, in finish
    func()
  File "/usr/lib/python3.11/site-packages/_pytest/fixtures.py", line 1037, in finish
    raise exc
  File "/usr/lib/python3.11/site-packages/_pytest/fixtures.py", line 1030, in finish
    func()
  File "/usr/lib/python3.11/site-packages/_pytest/fixtures.py", line 1037, in finish
    raise exc
  File "/usr/lib/python3.11/site-packages/_pytest/fixtures.py", line 1030, in finish
    func()
  File "/usr/lib/python3.11/site-packages/_pytest/fixtures.py", line 917, in _teardown_yield_fixture
    next(it)
  File "/jenkins/workspace/scylla-master/build/scylla/test/alternator/conftest.py", line 290, in test_table_s_forbid_rmw
    table.delete()
  File "/usr/lib/python3.11/site-packages/boto3/resources/factory.py", line 580, in do_action
    response = action(self, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/boto3/resources/action.py", line 88, in __call__
    response = getattr(parent.meta.client, operation_name)(*args, **params)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/client.py", line 530, in _api_call
    return self._make_api_call(operation_name, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/client.py", line 943, in _make_api_call
    http, parsed_response = self._make_request(
                            ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/client.py", line 966, in _make_request
    return self._endpoint.make_request(operation_model, request_dict)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/endpoint.py", line 119, in make_request
    return self._send_request(request_dict, operation_model)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/endpoint.py", line 202, in _send_request
    while self._needs_retry(
          ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/endpoint.py", line 354, in _needs_retry
    responses = self._event_emitter.emit(
                ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/hooks.py", line 412, in emit
    return self._emitter.emit(aliased_event_name, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/hooks.py", line 256, in emit
    return self._emit(event_name, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/hooks.py", line 239, in _emit
    response = handler(**kwargs)
               ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/retryhandler.py", line 207, in __call__
    if self._checker(**checker_kwargs):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/retryhandler.py", line 284, in __call__
    should_retry = self._should_retry(
                   ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/retryhandler.py", line 320, in _should_retry
    return self._checker(attempt_number, response, caught_exception)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/retryhandler.py", line 363, in __call__
    checker_response = checker(
                       ^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/retryhandler.py", line 247, in __call__
    return self._check_caught_exception(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/retryhandler.py", line 416, in _check_caught_exception
    raise caught_exception
  File "/usr/lib/python3.11/site-packages/botocore/endpoint.py", line 281, in _do_get_response
    http_response = self._send(request)
                    ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/endpoint.py", line 377, in _send
    return self.http_session.send(request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/botocore/httpsession.py", line 484, in send
    raise EndpointConnectionError(endpoint_url=request.url, error=e)
botocore.exceptions.EndpointConnectionError: Could not connect to the endpoint URL: "http://127.1.5.242:8000/"

Subprocess output:

Scylla version 5.4.0~dev-0.20230818.7275b8967cf5 with build-id 127b1fc7d4514ca9bfce896124c11cc600237136 starting ...
...
INFO  2023-08-18 04:29:30,662 [shard 0] schema_tables - Dropping keyspace alternator_alternator_Test_1692321709282
INFO  2023-08-18 04:29:30,663 [shard 0] schema_tables - Dropping alternator_alternator_Test_1692321709282.alternator_Test_1692321709282 id=9aca1430-3d65-11ee-8235-fab071cb0900 version=ab6ccb71-dd46-36cb-a087-bd59893ef6d1
INFO  2023-08-18 04:29:30,663 [shard 0] schema_tables - Dropping alternator_alternator_Test_1692321709282.alternator_Test_1692321709282_scylla_cdc_log id=9acfb980-3d65-11ee-8235-fab071cb0900 version=c9194819-7261-3e52-89d3-320ad6bc5a95
INFO  2023-08-18 04:29:30,664 [shard 0] database - Dropping alternator_alternator_Test_1692321709282.alternator_Test_1692321709282 without snapshot
INFO  2023-08-18 04:29:30,664 [shard 0] database - Dropping alternator_alternator_Test_1692321709282.alternator_Test_1692321709282_scylla_cdc_log without snapshot
INFO  2023-08-18 04:29:30,677 [shard 0] database - Truncating alternator_alternator_Test_1692321709282.alternator_Test_1692321709282_scylla_cdc_log without snapshot
INFO  2023-08-18 04:29:30,677 [shard 0] database - Truncating alternator_alternator_Test_1692321709282.alternator_Test_1692321709282 without snapshot
INFO  2023-08-18 04:29:30,887 [shard 0] alternator_ttl - sleeping 0.487 seconds until next period
INFO  2023-08-18 04:29:31,154 [shard 1] alternator_ttl - sleeping 0.492 seconds until next period
INFO  2023-08-18 04:29:31,392 [shard 0] alternator_ttl - sleeping 0.484 seconds until next period
Rate-limit: suppressed 45 backtraces on shard 0
INFO  2023-08-18 04:29:31,687 [shard 1] alternator_ttl - sleeping 0.46 seconds until next period
Reactor stalled for 49 ms on shard 1. Backtrace: 0xcbae463 0x3e82bb7 0x3e81fbf 0x3c75f9f 0x3c70253 0x3c6fcaf 0x3c70683 0x3c775bb 0x7db 0x42ccbeb 0x42ccb07 0x42cc923 0x42cbc6f 0x42c7713 0x42c694f 0x16507a1f 0x16505c9f 0x16504efb 0x164671ef 0x16466f6f 0x1647445b 0x1817b463 0x1817b19b 0x1817b00b 0x1817ad5f 0x1817aa87 0x1817aa13 0x1817a99b 0x1817a8f7 0xd093d83 0xd093baf 0xd093987 0xd0937cf 0xd09359f 0xd093523 0xd093297 0xd09303f 0xd092b27 0x3c97137 0x3c9fe43 0x3ca3a2f 0x3dd7c8b 0x3dd647f 0x3dd619b 0x3dd542b 0xdf24a53 0x39f4073 0x8bc73 0xf925b
INFO  2023-08-18 04:29:31,952 [shard 0] alternator_ttl - sleeping 0.432 seconds until next period
INFO  2023-08-18 04:29:32,163 [shard 1] alternator_ttl - sleeping 0.486 seconds until next period
INFO  2023-08-18 04:29:32,409 [shard 0] alternator_ttl - sleeping 0.476 seconds until next period
INFO  2023-08-18 04:29:32,659 [shard 1] alternator_ttl - sleeping 0.49 seconds until next period
INFO  2023-08-18 04:29:32,926 [shard 0] alternator_ttl - sleeping 0.472 seconds until next period
Reactor stalled for 81 ms on shard 1. Backtrace: 0xcbae463 0x3e82bb7 0x3e81fbf 0x3c75f9f 0x3c70253 0x3c6fcaf 0x3c70683 0x3c775bb 0x7db 0x21a0bcf3 0xcb6c537 0xcb6c3ab 0xcb6c03b 0xcb6dd07 0xcc09067 0xe168963 0x12557337 0x1255856b 0x125584ff 0x1255844b 0x12558407 0x125583db 0x1255832b 0x12555403 0x1255458b 0x12160bdb 0x1215da93 0x1322c243 0x13eed0ff 0x134dfc4f 0x134a87cf 0x13560e4f 0x13560ce3 0x1356099f 0x1356022f 0x1347263b 0x1342a6af 0x1359f04b 0x1359373b 0x13592303 0x13591d47 0x13579d8f 0x135787ff 0x13577d37 0x13577b1b 0x13577907 0x135776df 0x135774b3 0x13577303 0x135770d7 0x389e31b 0x42c95bb
Reactor stalled for 33 ms on shard 0. Backtrace: 0xcbae463 0x3e82bb7 0x3e81fbf 0x3c75f9f 0x3c70253 0x3c6fcaf 0x3c70683 0x3c775bb 0x7db 0x1111acd7 0x1111a0c3 0x111409d3 0x11140457 0x111402b7 0x1113f6f3 0x1113e2ab 0x1113d97b 0x111386df 0x11195803 0x111096ff 0x10ac43a7 0x18227d17 0x155787a7 0x155782b7 0x3c97137 0x3c9fe43 0x3ca3a2f 0x3ca1b2b 0x3705e8f 0x37035bb 0xcc4fb0f 0xdceea67 0xdcee62b 0xdcede3b 0xce7068f 0xcc4d4a7 0x30597 0x3066f 0xcb6842f
INFO  2023-08-18 04:29:33,163 [shard 1] alternator_ttl - sleeping 0.492 seconds until next period
INFO  2023-08-18 04:29:33,429 [shard 0] alternator_ttl - sleeping 0.47 seconds until next period
INFO  2023-08-18 04:29:33,682 [shard 1] alternator_ttl - sleeping 0.492 seconds until next period
Reactor stalled for 39 ms on shard 0. Backtrace: 0xcbae463 0x3e82bb7 0x3e81fbf 0x3c75f9f 0x3c70253 0x3c6fcaf 0x3c70683 0x3c775bb 0x7db 0x111cd09f 0x111cb7cf 0x111cb0cf 0x111b8f53 0x1119552b 0x111096ff 0x10ac43a7 0x18227d17 0x155787a7 0x155782b7 0x3c97137 0x3c9fe43 0x3ca3a2f 0x3ca1b2b 0x3705e8f 0x37035bb 0xcc4fb0f 0xdceea67 0xdcee62b 0xdcede3b 0xce7068f 0xcc4d4a7 0x30597 0x3066f 0xcb6842f
INFO  2023-08-18 04:29:34,021 [shard 0] alternator_ttl - sleeping 0.417 seconds until next period
INFO  2023-08-18 04:29:34,216 [shard 1] alternator_ttl - sleeping 0.459 seconds until next period
Reactor stalled for 36 ms on shard 0. Backtrace: 0xcbae463 0x3e82bb7 0x3e81fbf 0x3c75f9f 0x3c70253 0x3c6fcaf 0x3c70683 0x3c775bb 0x7db 0x10fb9b7b 0x10fb9b37 0x10fb4dd7 0x10fb9ae3 0x10fcc50b 0x10fbc1ff 0x10fd7e2f 0x10fd7acb 0x10fd7923 0x10fd75d3 0x10fd73af 0x10fd700b 0x10ecfa2b 0x10cc8da3 0x101ddf7b 0x101ddf03 0x101dde93 0x11342887 0x1131da07 0x1128db07 0x10ad6b7b 0x10acb4d3 0x10acaedf 0x10cdf273 0x101c89db 0x101c75df 0x101cc35f 0x101cc047 0x101cbedf 0x101c6a2f 0x101c592f 0x101c505f 0x101c4eeb 0x101c4b8b 0x101c4403 0x101c3d1b 0x101c32bb 0x101cd72b 0x101ccf6f 0x3c97137 0x3c9fe43 0x3ca3a2f 0x3ca1b2b 0x3705e8f 0x37035bb 0xcc4fb0f 0xdceea67 0xdcee62b 0xdcede3b 0xce7068f 0xcc4d4a7 0x30597 0x3066f 0xcb6842f
INFO  2023-08-18 04:29:34,458 [shard 0] alternator_ttl - sleeping 0.481 seconds until next period
INFO  2023-08-18 04:29:34,709 [shard 1] alternator_ttl - sleeping 0.492 seconds until next period
INFO  2023-08-18 04:29:34,974 [shard 0] alternator_ttl - sleeping 0.467 seconds until next period
AddressSanitizer:DEADLYSIGNAL
=================================================================
==1522==ERROR: AddressSanitizer: SEGV on unknown address 0xfffffffffffffff8 (pc 0xffff8b6dd174 bp 0xffffc84fb830 sp 0xffffc84fb830 T0)
==1522==The signal is caused by a READ memory access.
AddressSanitizer:DEADLYSIGNAL
AddressSanitizer: nested bug in the same thread, aborting.
Alternator tests failure
=== TEST.PY SUMMARY START ===
Test exited with code 1

=== TEST.PY SUMMARY END ===
```

The dereferenced address of -8 (0xfffffffffffffff8) is the same as in https://github.com/scylladb/scylladb/issues/14926
so it might be a duplicate of the same bug.

### LINKED ISSUE #15389: ARM: topology_experimental_raft: test_joining_old_node_fails in debug mode
`test_raft_cluster_features.test_joining_old_node_fails` failed on ARM in debug mode with error:

```
_________________________ test_joining_old_node_fails __________________________

manager = <test.pylib.manager_client.ManagerClient object at 0xfffeff6ca650>

    @pytest.mark.asyncio
    async def test_joining_old_node_fails(manager: ManagerClient) -> None:
        for _ in range(3):
            await manager.server_add()
>       await test_cluster_features.test_joining_old_node_fails(manager)

test/topology_experimental_raft/test_raft_cluster_features.py:38: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
test/topology/test_cluster_features.py:131: in test_joining_old_node_fails
    await change_support_for_test_feature_and_restart(manager, servers, enable=True)
test/topology/test_cluster_features.py:59: in change_support_for_test_feature_and_restart
    await asyncio.gather(*(manager.server_start(srv.server_id, expected_error) for srv in srvs))
test/pylib/manager_client.py:155: in server_start
    await self.client.get_text(f"/cluster/server/{server_id}/start", params=params)
test/pylib/rest_client.py:82: in get_text
    ret = await self._fetch("GET", resource_uri, response_type = "text", host = host,
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <test.pylib.rest_client.UnixRESTClient object at 0xfffeff679ad0>
method = 'GET', resource = '/cluster/server/434/start', response_type = 'text'
host = None, port = None, params = None, json = None, timeout = None

    async def _fetch(self, method: str, resource: str, response_type: Optional[str] = None,
                     host: Optional[str] = None, port: Optional[int] = None,
                     params: Optional[Mapping[str, str]] = None,
                     json: Optional[Mapping] = None, timeout: Optional[float] = None) -> Any:
        # Can raise exception. See https://docs.aiohttp.org/en/latest/web_exceptions.html
        assert method in ["GET", "POST", "PUT", "DELETE"], f"Invalid HTTP request method {method}"
        assert response_type is None or response_type in ["text", "json"], \
                f"Invalid response type requested {response_type} (expected 'text' or 'json')"
        # Build the URI
        port = port if port else self.default_port if hasattr(self, "default_port") else None
        port_str = f":{port}" if port else ""
        assert host is not None or hasattr(self, "default_host"), "_fetch: missing host for " \
                "{method} {resource}"
        host_str = host if host is not None else self.default_host
        uri = self.uri_scheme + "://" + host_str + port_str + resource
        logging.debug(f"RESTClient fetching {method} {uri}")
    
        client_timeout = ClientTimeout(total = timeout if timeout is not None else 300)
        async with request(method, uri,
                           connector = self.connector if hasattr(self, "connector") else None,
                           params = params, json = json, timeout = client_timeout) as resp:
            if resp.status != 200:
                text = await resp.text()
>               raise HTTPError(uri, resp.status, params, json, text)
E               test.pylib.rest_client.HTTPError: HTTP error 500, uri: http+unix://api/cluster/server/434/start, params: None, json: None, body:
E               failed to start the node, server_id 434, IP 127.192.14.35, workdir scylla-434, host_id 782285f8-d996-4859-903c-b04e092bf49a, cql [not connected]
E               Check the log files:
E               /jenkins/workspace/scylla-enterprise/build/scylla/testlog/aarch64/test.py.debug-release-dev.log
E               /jenkins/workspace/scylla-enterprise/build/scylla/testlog/aarch64/debug/scylla-434.log

test/pylib/rest_client.py:69: HTTPError
```

link to failure:
https://jenkins.scylladb.com/job/scylla-enterprise/job/build/1017/artifact/testlog/aarch64/debug/topology_experimental_raft.test_raft_cluster_features.1.log

### PR DESCRIPTION
**Pull request by gusev-p**
The stall detector uses glibc backtrace function to collect backtraces, this causes ASAN failures on ARM. For now we just disable the stall detector in this configuration, the ticket about migrating
to libunwind: scylladb/seastar#1878

Fixes #15389
Fixes #15090

### CODE REVIEW

File: main.cc [hunk 1]
```diff
@@ -691,7 +691,7 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             // disable reactor stall detection during startup
             auto blocked_reactor_notify_ms = engine().get_blocked_reactor_notify_ms();
             smp::invoke_on_all([] {
-                engine().update_blocked_reactor_notify_ms(std::chrono::milliseconds(1000000));
+                engine().update_blocked_reactor_notify_ms(std::chrono::milliseconds(1000000000));
```
Comment (kbr-scylla) [inline main.cc:694]: Can we use something more readable like `hours(1)`?

File: main.cc [hunk 2]
```diff
@@ -1026,10 +1026,15 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             db.local().init_commitlog().get();
             db.invoke_on_all(&replica::database::start).get();
 
+// The stall detector uses glibc backtrace function to
```
Comment (kbr-scylla) [inline main.cc:1029]: Please add `FIXME:` prefix

File: main.cc (hunk 1 above)
Comment (avikivity) [reply main.cc:694]: Or `1h`

Comment (avikivity) [reply main.cc:694]: milliseconds(1000000) could be written 1000s, no need to cast.

File: main.cc (hunk 2 above)
Comment (gusev-p) [reply main.cc]: done

File: main.cc (hunk 1 above)
Comment (gusev-p) [reply main.cc]: done

**Review by avikivity — APPROVED** [review]

**Comment by yaronkaikov** [comment]
It seems that this commit break `build` in `aarch64` - https://jenkins.scylladb.com/job/scylla-master/job/build/1659/
```
[2023-10-17T00:21:33.913Z] [411/3101] CXX build/debug/main.o
[2023-10-17T00:21:33.913Z] FAILED: build/debug/main.o 
[2023-10-17T00:21:33.913Z] clang++ -MD -MT build/debug/main.o -MF build/debug/main.o.d -I/jenkins/workspace/scylla-master/build/scylla/seastar/include -I/jenkins/workspace/scylla-master/build/scylla/build/debug/seastar/gen/include -U_FORTIFY_SOURCE -Werror=unused-result -fsanitize=address -fsanitize=undefined -fno-sanitize=vptr -DSEASTAR_API_LEVEL=7 -DSEASTAR_BUILD_SHARED_LIBS -DSEASTAR_SSTRING -DSEASTAR_SCHEDULING_GROUPS_COUNT=16 -DSEASTAR_DEBUG -DSEASTAR_DEFAULT_ALLOCATOR -DSEASTAR_SHUFFLE_TASK_QUEUE -DSEASTAR_DEBUG_SHARED_PTR -DSEASTAR_LOGGER_TYPE_STDOUT -DSEASTAR_TYPE_ERASE_MORE -DBOOST_NO_CXX98_FUNCTION_BASE -DFMT_SHARED -I/usr/include/p11-kit-1   -ffile-prefix-map=/jenkins/workspace/scylla-master/build/scylla=. -march=armv8-a+crc+crypto -DDEBUG -DSANITIZE -DDEBUG_LSA_SANITIZER -DSCYLLA_ENABLE_ERROR_INJECTION -O0 -DSCYLLA_BUILD_MODE=debug -g -gz -iquote. -iquote build/debug/gen --std=gnu++20  -ffile-prefix-map=/jenkins/workspace/scylla-master/build/scylla=. -march=armv8-a+crc+crypto  -DBOOST_TEST_DYN_LINK   -DNOMINMAX -DNOMINMAX -fvisibility=hidden -Wall -Werror -Wimplicit-fallthrough -Wno-mismatched-tags -Wno-c++11-narrowing -Wno-overloaded-virtual -Wno-unused-command-line-argument -Wno-unsupported-friend -Wno-implicit-int-float-conversion -Wno-psabi -Wno-narrowing -Wno-error=deprecated-declarations -DXXH_PRIVATE_API -DSEASTAR_TESTING_MAIN -DFMT_DEPRECATED_OSTREAM  -c -o build/debug/main.o main.cc
[2023-10-17T00:21:33.914Z] main.cc:692:18: error: unused variable 'blocked_reactor_notify_ms' [-Werror,-Wunused-variable]
[2023-10-17T00:21:33.914Z]             auto blocked_reactor_notify_ms = engine().get_blocked_reactor_notify_ms();
[2023-10-17T00:21:33.914Z]                  ^
```
@gusev-p @avikivity @kbr-scylla

**Comment by gusev-p** [comment]
> It seems that this commit break `build` in `aarch64` - https://jenkins.scylladb.com/job/scylla-master/job/build/1659/

How it's possible? Why CI doesn't build on ARM?

**Comment by yaronkaikov** [comment]
We currently don't run `ARM` during CI or Gating

**Comment by mykaul** [comment]
> We currently don't run `ARM` during CI or Gating

Especially as it fails (due to items fixed in this PR and others). But I think build should be running.

**Comment by yaronkaikov** [comment]
> > We currently don't run `ARM` during CI or Gating
> 
> Especially as it fails (due to items fixed in this PR and others). But I think build should be running.

Btw , those kind of changes can be tested with https://jenkins.scylladb.com/job/scylla-master/job/byo/job/byo_build_tests_dtest/ ( it's already support arm for build and unit test)

**Pull request closed without merging**